pkg/base1/test-user.js

```diff
@@ -22,6 +22,9 @@ QUnit.test("user object", async assert => {
     assert.equal(typeof user.shell, "string", "user shell");
     assert.equal(typeof user.home, "string", "user home");
     assert.equal(typeof user.id, "number", "user id");
+    if (await mock_info("pybridge")) {
```

Comment:
If that works, that leaves me very worried -- that test [failed in the pytest scenario](https://github.com/cockpit-project/cockpit/actions/runs/9256807304/job/25463602338?pr=20517#step:2:811), not the "C unit tests" one. So would that mean that we are running these tests against the C bridge somehow?

Reply:
Ah no, it [failed again](https://github.com/cockpit-project/cockpit/actions/runs/9281787892/job/25538459894?pr=20517#step:2:811), so that's not it.